create-table-as.md

```diff
@@ -1,18 +1,34 @@
 ---
 title: CREATE TABLE AS
-summary: The CREATE TABLE AS statement creates a new table from the results of a SELECT statement.
+summary: The CREATE TABLE AS statement persists the result of a query into the database for later reuse.
 toc: false
 ---
 
-The `CREATE TABLE AS` statement creates a new table from the results of a [`SELECT`](select.html) statement.
-
-These tables are not designed for long-term use, as they do not support some common table features like [Primary Keys](primary-key.html) and [interleaving](interleave-in-parent.html). For similar functionality with more robust feature support, [create a table](create-table.html) and then use [`INSERT INTO SELECT`](insert.html#insert-from-a-select-statement).
+The `CREATE TABLE ... AS` statement creates a new table from the results of a query.
 
 <div id="toc"></div>
 
+## Intended use
+
+Tables created with `CREATE TABLE ... AS` are intended to persist the
+result of a query for later reuse.
+
+This can be more efficient than a [view](create-view.html) when the following two conditions are met:
+- the result of the query is used as-is multiple times;
+- the copy needs not be kept up-to-date with the original table over time.
+
+When the results of a query are reused multiple times within a larger
+query, a view is advisable instead: the query optimizer can "peek"
+into the view and optimize the surrounding query using the primary key
+and indices of the tables mentioned in the view query.
+
+A view is also advisable when the results must be up-to-date: q view
```

Comment:
Colon should be a semicolon. But feel free to use a period instead.

Also, `q` should be `A`.